packages/shared.test/src/handlers/util/handleOpenApiCall.ts

```diff
@@ -59,7 +59,7 @@ export type ExtractBody<
 
 export type ExtractResponse<
   X extends ((...args: any[]) => Promise<any>),
-> = Awaited<ReturnType<X>> & {};
+> = Awaited<ReturnType<X>>;
```

Comment:
Had to remove this so we could allow `|undefined` types in return, like we do for geotime series points, because we could fetchLatestPoint on a track with no data, which the API would return no data for.